src/Core/Content/ImportExport/ImportExportProfileTranslationEntity.php

```diff
@@ -3,8 +3,12 @@
 namespace Shopware\Core\Content\ImportExport;
 
 use Shopware\Core\Framework\DataAbstractionLayer\TranslationEntity;
+use Shopware\Core\Framework\Feature;
 use Shopware\Core\Framework\Log\Package;
 
+/**
+ * @deprecated tag:v6.8.0 - reason:remove-entity - Will be removed
```

Comment:
Already adding a DB migration for dropping the table destructively in the 6_8 folder would be great, so it can't be forgotten